src/components/radio-button-list-item/styles/SRadioButtonListItem.ts

```diff
@@ -13,6 +13,18 @@ const Bronze = css`
   cursor: pointer;
 
   ${CSSProgressiveBody03Medium};
+  
+  *{
+    flex: 0 0 auto;
+  }
+  
+  [data-main]{
```

Comment:
Let's create this as an Interface component like RadioButtonListItemContent and ListItemContent, in both places. This is going to be hard to communicate and not sure if others will notice this, I think this will lead to the original problem of having p tags with inline style all over the place.

Reply:
Done